h2o-core/src/main/java/water/util/SBPrintStream.java

```diff
@@ -208,6 +208,74 @@ public SBPrintStream toJavaStringInit(float[] ss) {
     }
     return p('}');
   }
+  
+  private final static int METHOD_SIZE_LIMIT = 10240;
```

Comment:
java method size limit is 64K of bytecode, 10k array elements has proven to be a good size for split